app/controllers/reviews_controller.rb

```diff
@@ -0,0 +1,32 @@
+class ReviewsController < ApplicationController
+  def index
+  end
+  def show
+    @review = Review.find(params[:id])
+  end
+  def new
```

Comment:
Gonna have to agree with the Hound on this -- this just looks busy and unreadable, to me.